terraform/aws/analytical-platform-data-production/tooling-iam/transcribe-iam.tf

```
@@ -0,0 +1,60 @@
+data "aws_iam_policy_document" "transcribe_service_access" {
+  #checkov:skip=CKV_AWS_356: skip requires access to multiple resources
+  statement {
+    sid    = "transcribeGenericAccess"
+    effect = "Allow"
+    actions = [
+      "transcribe:List*",
+      "transcribe:Get*",
+      "transcribe:Describe*",
+      "transcribe:StartStreamTranscription",
+      "transcribe:StartStreamTranscriptionWebSocket",
+    ]
+    resources = ["*"]
+  }
+  statement {
+    sid    = "transcribeServiceAccess"
+    effect = "Allow"
+    actions = [
+      "transcribe:GetTranscriptionJob",
+      "transcribe:GetVocabulary",
+      "transcribe:CreateVocabulary",
+      "transcribe:StartTranscriptionJob",
+      "transcribe:DeleteTranscriptionJob",
+      "transcribe:UpdateVocabulary",
+      "transcribe:CreateLanguageModel",
+    ]
+    resources = [
+      "arn:aws:transcribe:eu-west-1:${var.account_ids["analytical-platform-data-production"]}:vocabulary/*",
+      "arn:aws:transcribe:eu-west-1:${var.account_ids["analytical-platform-data-production"]}:transcription-job/*",
+      "arn:aws:transcribe:eu-west-1:${var.account_ids["analytical-platform-data-production"]}:language-model/*"
```

```suggestion
      "arn:aws:transcribe:eu-west-*:${var.account_ids["analytical-platform-data-production"]}:vocabulary/*",
      "arn:aws:transcribe:eu-west-*:${var.account_ids["analytical-platform-data-production"]}:transcription-job/*",
      "arn:aws:transcribe:eu-west-*:${var.account_ids["analytical-platform-data-production"]}:language-model/*"
```

Do we want/need to limit this to ireland?

---

yes we can use the region from the provider because the default provider is eu-west-1

---

Changed to get region from `${data.aws_region.current.region}`